resources/artifact-ignores.properties

```diff
@@ -809,3 +809,4 @@ xframium = https://www.jenkins.io/security/plugins/#suspensions
 
 # https://jenkins.io/security/advisory/2022-11-15/
 config-rotator = https://www.jenkins.io/security/plugins/#suspensions
+lightstep-incident-response-plugin
```

Comment:
```suggestion
lightstep-incident-response
```

Please also add a URL providing information about the deprecation/removal for users of this plugin, as demonstrated by other entries in this file.